ts/data/data.ts

```diff
@@ -551,6 +576,42 @@ async function removeAllMessagesInConversation(conversationId: string): Promise<
   );
 }
 
+async function findAllMessageFromSendersInConversation(
+  args: FindAllMessageFromSendersInConversationTypeArgs
+): Promise<Array<MessageModel>> {
+  const msgAttrs = await channels.findAllMessageFromSendersInConversation(args);
+
+  if (!msgAttrs || isEmpty(msgAttrs) || !isArray(msgAttrs)) {
+    return [];
+  }
+
+  return msgAttrs.map((msg: any) => new MessageModel(msg));
+}
+
+async function findAllMessageHashesInConversation(
```

Comment:
Shouldn't the return type just be an array of strings here?

Reply:
well, we need the message models as we will call markAsDeleted on it, and there is no easy way to do it on the node side currently.
![image](https://github.com/user-attachments/assets/887bf4d6-ccae-433c-abd4-e880f3d32e52)

Reply:
Cool. Resolving